src/graph.js

```diff
@@ -1,7 +1,61 @@
-/* eslint-disable */
 class Graph {
   constructor() {
-    this.graph = {};
+    this.graph = [];
+  }
+  addNode(value, edge) {
+    const cgl = this.graph.length;
+    const node = { value, edges: [] };
+    if (cgl === 1) {
+      node.edges.push(this.graph[0].value);
+      this.graph[0].edges.push(node.value);
+    } else if (cgl > 1 && edge) {
+      node.edges.push(edge);
+    }
+    this.graph.push(node);
+  }
+  getNode(node) {
+    const cgl = this.graph.length;
+    for (let i = 0; i < cgl; i++) {
+      if (this.graph[i].value === node) return this.graph[i];
+    } return null;
```

Comment:
As part of syntax convention, you should move this return statement down to the next line.